### PR TITLE
Remove parameters from sonata admins

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC break.

## Changelog

```markdown
### Removed
- All container parameters that were used to customize classes.
```

### CODE REVIEW

**Review by VincentLanglet — DISMISSED** [review]
Should we add this to the upgrade note ?

**Review by phansys — COMMENTED** [review]
I am not up to date with these changes.
Is there a proposed replacement for these configuration points?
Do we have documentation regarding this?

**Comment by jordisala1991** [comment]
This extension points are not documented and there is not proposed replacement. They are dangerous because no one is controlling what you can define on this parameters. On config you get nice validation and restrictions.

The idea is to override the service definition if you want to change something or open an issue if you need some extension point, to discuss if it is really worth it or it can be done in another way.

**Comment by jordisala1991** [comment]
Added the UPGRADE note.

**Review by VincentLanglet — DISMISSED** [review]

File: UPGRADE-4.0.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+### Container parameters
+
+  We have been using container parameters on SonataMediaBundle 3.x as an undocumented extension
+  points for things like classes, and configurations.
```
Comment (phansys) [inline UPGRADE-4.0.md:7]:
```suggestion
  We have been using container parameters as undocumented extension points for things like classes and configurations.
```

File: UPGRADE-4.0.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+
+  In SonataMediaBundle 4.0 those are completely removed and we encourage you to use the default
+  dependency injection override to change services configuration if you need to.
```
Comment (phansys) [inline UPGRADE-4.0.md:10]:
```suggestion
  dependency injection override to change the default values for the removed service configurations if you need to.
```

**Review by phansys — DISMISSED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Review by phansys — APPROVED** [review]

**Comment by phansys** [comment]
Thank you @jordisala1991.

**Pull request merged**